src/extensions/ReputationFaucet.sol

```diff
@@ -0,0 +1,24 @@
+pragma solidity ^0.4.13;
+
+import 'ROOT/Controller.sol';
+
+
+// FIXME this can be removed once the contract is implemented in Solidity
+contract Branch {
+    function stub () {}
+}
+
+
+contract ReputationFaucet is Controlled {
```

Comment:
I think we might be able to just delete this contract now?  It doesn't actually do anything (and hasn't since reporting branch was merged in).